mmcv/runner/hooks/evaluation.py

```diff
@@ -344,7 +344,10 @@ def _save_ckpt(self, runner, key_score):
             runner.meta['hook_msgs']['best_ckpt'] = self.best_ckpt_path
 
             runner.save_checkpoint(
-                self.out_dir, best_ckpt_name, create_symlink=False)
+                self.out_dir, 
+                filename_tmpl=f'best_{self.key_indicator}_' + cur_type + '_{}.pth', 
```

Comment:
Should we directly pass the `best_ckpt_name` to the `filename_tmpl`?
```suggestion
                filename_tmpl=best_ckpt_name, 
```

Reply:
Yes we could, but, might including iter number in the filename be a good option?In my opinion I would prefer to see iteration number in the filename

Reply:
The original `best_ckpt_name`  has contained the current epoch or iteration.